Tools/CISupport/ews-build/steps.py

```diff
@@ -120,10 +126,11 @@ def commit_status_url(cls, sha, repository_url=None):
         return '{}/statuses/{}'.format(api_url, sha)
 
     @classmethod
-    def credentials(cls):
+    def credentials(cls, user=None):
         try:
             passwords = json.load(open('passwords.json'))
```

Comment:
Do we want to cache this data or open this file for every api call?

Reply:
Good point, will cache that